airbyte-integrations/connectors/source-greenhouse/source_greenhouse/manifest.yaml

```diff
@@ -61,23 +61,29 @@ definitions:
       $ref: "#/definitions/retriever"
       requester: "#/definitions/requester"
     incremental_sync:
-      type: CustomIncrementalSync
-      class_name: source_greenhouse.components.GreenHouseSlicer
-      request_cursor_field: "updated_after"
-      cursor_field: "updated_at"
+      type: DatetimeBasedCursor
+      cursor_field: "{{ parameters.get('cursor_field', 'updated_at') }}"
+      cursor_datetime_formats:
+        - "%Y-%m-%dT%H:%M:%S.%_msZ"
+      datetime_format: "%Y-%m-%dT%H:%M:%S.%_msZ"
+      start_datetime:
+        type: MinMaxDatetime
+        datetime: "1970-01-01T00:00:00.000Z"
+        datetime_format: "%Y-%m-%dT%H:%M:%S.%_msZ"
+      start_time_option:
+        type: RequestOption
+        inject_into: request_parameter
+        field_name: "{{ parameters.get('cursor_request_option', 'updated_after') }}"
   applications_stream:
     $ref: "#/definitions/base_incremental_stream"
     $parameters:
       name: "applications"
       path: "applications"
+      cursor_field: "applied_at"
+      cursor_request_option: "created_after"
```

Comment:
small suggestion, instead of adding these two arbitrary parameters that we then need to account for using `field_name: "{{ parameters.get('cursor_request_option', 'updated_after')`, I would suggest that we instead just define a new:
```
incremental_sync:
  type: DatetimeBasedCursor
  ...
  start_time_option:
    type: RequestOption
    inject_into: request_parameter
    field_name: "created_after"
```

I think that reads more easily than the extra parameter injection because at first I thought it was named wrong as `cursor_request_option`. wdyt?

Reply:
removed `cursor_request_option` and `cursor_field` from parameters. updated `incremental_sync` to use $ref to basic incremental_sync instead and overrided `start_time_option` and `cursor_field`